### DISCUSSION TITLE
Interacting with added latex

### DISCUSSION THREAD

**Discussion by dryya** [xournalpp/xournalpp#4500] [Q&A] [answered]
Hi there,

Apologies if this has an easy answer. After a lot of searching, I can't seem to find information on what should be very simple: interacting with added latex in any way at all (e.g. moving it around, editing it). To be clear, latex works fine for me: it brings up the menu, renders an image, and puts it in the file. I can even move it around at that point. But as soon as I deselect it, it's like it's baked into the background PDF and I can't do anything with it. How do I select it again?

**Comment by rolandlo** [comment, marked as answer]
Use the selection tool to select the LaTeX formula and then double-click on it to open up the LaTeX dialog where you can modify the formula.

**Reply by dryya** [reply]
Thank you! for some reason I was convinced the hand tool was the selection tool, and hadn't found the drop-down menu which let me switch from the "play object" to "select object" tool.
